kong/llm/drivers/shared.lua

```diff
@@ -219,7 +243,7 @@ end
 -- @param {string} frame input string to format into SSE events
 -- @param {boolean} raw_json sets application/json byte-parser mode
 -- @return {table} n number of split SSE messages, or empty table
-function _M.frame_to_events(frame, raw_json_mode)
```

Comment:
Are you sure that you patched all `frame_to_events` invocations. This is a shared library and there may be some false expectations when the function signature changes.

Reply:
Yeah checked all providers. This won't happen again after this release, will be moved into their respective drivers.

I have no idea why I put this function in `shared` in the first place...